CHANGELOG.md

```diff
@@ -18,6 +18,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Added highlighted filter to search endpoint - [#57](https://github.com/DigitalExcellence/dex-backend/issues/57)
 - Setup basic unit test framework to ensure that the core functionality of the application works - [#65](https://github.com/DigitalExcellence/dex-backend/issues/65)
 - Added roles and authorization validation. - [#107](https://github.com/DigitalExcellence/dex-backend/issues/107)
+- Added Wizard Controller and service - [#127](https://github.com/DigitalExcellence/dex-backend/issues/127)
+- Added Wizard GitLab metadata - [#125](https://github.com/DigitalExcellence/dex-backend/issues/125)
+- Added Wizard GitHub Metadata - [#126](https://github.com/DigitalExcellence/dex-backend/issues/126)
```

Comment:
This is not true since the GitHub metadata is handled in the front end as discussed in a previous thread in this PR.

Reply:
i removed the entry